analyses/uga/docs/uga_flood_risk.Rmd

```diff
@@ -0,0 +1,266 @@
+knitr::opts_chunk$set(message = FALSE) # do not print messages by default
+knitr::opts_chunk$set(warning = FALSE) # do not print warnings by default
+
+options(scipen = 999) # turn off scientific notation
+options(encoding = "UTF-8") # set encoding to UTF-8 instead of ANSI
+
+# paths
+data_dir <- Sys.getenv("AA_DATA_DIR") 
+public_raw_uga_dir <- paste0(data_dir,'/public/raw/uga')
+private_exploration_uga_dir <- paste0(data_dir,'/private/exploration/uga')
+
+uga_adm1_shapefile_path <- paste0(public_raw_uga_dir, "/cod_ab/uga_admbnda_ubos_20200824_shp/uga_admbnda_adm1_ubos_20200824.shp")
+uga_adm3_shapefile_path <- paste0(public_raw_uga_dir, "/cod_ab/uga_admbnda_ubos_20200824_shp/uga_admbnda_adm3_ubos_20200824.shp")
+
+ec_jrc_image_path <- paste0(private_exploration_uga_dir, "/ec_jrc/ec_jrc_uga_flood_risk_adm3_20220302.png")
+thinkhdz_urban_path <- paste0(public_raw_uga_dir, "/thinkhazard/urban_flood.png")
+thinkhzd_river_path <- paste0(public_raw_uga_dir, "/thinkhazard/river_flood.png")
+thinkhzd_legend_path <- paste0(public_raw_uga_dir, "/thinkhazard/legend_horizontal.png")
+
+pop_path <- paste0(public_raw_uga_dir, "/cod_ps/uga_admpop_2020_csv/uga_admpop_adm2_2020proj_1y.csv")
+
+displ_path <- paste0(public_raw_uga_dir, "/idmc/IDMC_GIDD_disasters_internal_displacement_data_2020-1646173891391.csv")
+
+icpac_image_path <- paste0(private_exploration_uga_dir, "/icpac/forecast_adm1_overlay_20220301.png")
+icpac_data_path <- paste0(private_exploration_uga_dir, "/icpac/stats_summary_20220301.csv")
+unma_image_path <- paste0(public_raw_uga_dir, "/unma/seasonal-forecast-mam2022.gif")
+```
+
+```{r libraries, include = FALSE}
+packages <- c('tidyverse', 'ggthemes', 'kableExtra', 'knitr', 'flextable', 'rebus')
+
+# install packages not yet installed
+installed_packages <- packages %in% rownames(installed.packages())
+
+if (any(installed_packages == FALSE)) {
+  install.packages(packages[!installed_packages])
+}
+
+# load libraries 
+lapply(packages, library, character.only = TRUE)
+
+## If collapse_rows() doesn't work, this manual install will fix the issue until kabelExtra issues an update: devtools::install_github(repo="haozhu233/kableExtra", ref="a6af5c0")
+```
+
+
+<br>
+<br>
+<br>
+
+# Uganda Flood Risk MAM 2022
+Analysis last updated on: `r Sys.Date()`
+
+This report compiles data on the risk of flood and its potential humanitarian impact. Administrative boundaries as of August 2020.
+
+***
+
+## Flood Risk
+
+```{r flood_risk, include = FALSE, message = FALSE, warning = FALSE}
+#source("../notebooks/process_flood_risk_data.R") # removed to knit. Script produces dataframe and png of flood risk included in this report.
+
+```
+The map below depicts flood prone areas for flood events with 10-year return period, according to the European Commission Joint Research Council. Resolution is approx. 1km and colors indicate water depth (in m). 
+
+&nbsp;
+
+![Source: EC-JRC](`r ec_jrc_image_path`)
```

Comment:
Not going to add it now because it was already sent to the requesting office. But higher water levels = more severe flood. The areas on the map are those expected to be flooded in a 10-year flood.